rs/ic_os/config/src/config_ini.rs

```diff
@@ -9,8 +9,7 @@ use anyhow::{Context, Result};
 
 pub type ConfigMap = HashMap<String, String>;
 pub struct ConfigIniSettings {
-    pub ipv6_prefix: Option<String>,
-    pub ipv6_address: Option<Ipv6Addr>,
```

Comment:
nit: Could you also remove https://github.com/dfinity/ic/blob/2837ea35bfb5d5d034f927d245f796c1bee2804f/ic-os/components/early-boot/setup-hostname/hostos/setup-hostname.sh#L64

Reply:
Good catch! Removed!